src/web/app/pages-help/instructor-help-page/instructor-help-page.component.html

```diff
@@ -1,49 +1,65 @@
-<div>
-  <h1 class="color-orange"> Help for Instructors</h1>
-  <div>
-    <p>
-      Have questions about how to use TEAMMATES? This page answers some frequently asked questions.
-      <br> If you are new to TEAMMATES, our <a routerLink="../getting-started">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
-      <br> If you have any remaining questions, don't hesitate to <a href="mailto:{{ supportEmail }}">email us</a>. We respond within 24 hours.
-    </p>
-    <div id="topics" style="display: block;">
+<div class="main-body" #helpPage>
+  <div id={{Sections.body}}>
```

Comment:
Wrap the value in `""`, i.e. this should be `id="{{ Sections.body }}"`. Similarly for others.

Reply:
Gotcha